modules/mod_ls.c

```diff
@@ -633,6 +633,27 @@ static int listfile(cmd_rec *cmd, pool *p, const char *resp_code,
       t = pr_localtime(p, (time_t *) &sort_time);
     }
 
+    unsigned char *emulate_windows = NULL;
+    emulate_windows = get_param_ptr(TOPLEVEL_CONF, "EmulateWindows", FALSE);
```

Comment:
FYI, I strive to ensure that the ProFTPD code compiles cleanly as ANSI code; this means (among other things) that all variable declarations must be at the start of the code block, rather than in the middle.  So I might adjust this particular coding for that; just a head's up.